subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/bintray/ShipkitBintrayPlugin.java

```diff
@@ -55,6 +57,17 @@ public void apply(final Project project) {
             LOG.lifecycle(welcomeMessage);
         });
 
+        bintrayUpload.doLast(task -> {
+                BintrayUploadTask bintrayUploadTask = (BintrayUploadTask) task;
+                if (bintrayUploadTask.isEnabled() && hasNoArtifacts(bintrayUploadTask)) {
+                    throw new GradleException("'" + bintrayUploadTask.getName() + "' currently has no artifacts for publication.\n" +
+                        " - if the current project '" + project.getPath() + "' is not supposed to publish to bintray -> let's disable the bintray task (bintrayUpload.enabled = false)\n" +
```

Comment:
"let's disable the bintray task" reads somewhat odd. I suggest "please disable the bintray task"